uui-components/src/table/DataTableCellContainer.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { cx, DataColumnProps, IHasCX, uuiDataTableHeaderCell } from '@epam/uui-core';
+import { FlexCell } from '../layout';
+import css from './DataTableCellContainer.module.scss';
+ 
+/**
+ * Header cell props of DataTable with Timeline.
+ */
+export interface DataTableCellContainerProps extends IHasCX, React.PropsWithChildren {
+    /** DataColumnProps object for the column the cell is at */
+    column: DataColumnProps;
+}
+ 
+export function DataTableCellContainer(props: DataTableCellContainerProps) {
+    return (
+        <FlexCell
+            { ...props.column }
+            minWidth={ props.column.width }
+            rawProps={ {
+                role: 'columnheader',
+                'aria-sort': 'none',
```

Comment:
I guess we should remove 'aria-sort' from here, because container don't know if any sorting is applied or not. And we should add `rawProps` prop and spread it here, to be able to pass such attributes from `DataTableCellContainer` consumers.